src/Modal/__tests__/Modal.test.tsx

```diff
@@ -1,15 +1,19 @@
 import ReactModal from 'react-modal';
 import { Global, css } from '@emotion/react';
 import { shallow } from 'enzyme';
+import { mocked } from 'ts-jest/utils';
 
+import theme from '../../theme';
 import buildModalCss from '../buildModalCss';
 import { Modal } from '../Modal';
 
 jest.mock('../buildModalCss');
 
+const isOpen = false;
+
 describe('<Modal />', () => {
   it('renders a Global for the modal styles', () => {
-    const modal = shallow(<Modal>My Modal</Modal>);
+    const modal = shallow(<Modal {...{ isOpen, theme }}>My Modal</Modal>);
```

Comment:
`isOpen` and `theme` are required props; this just wasn't sufficiently expressed via PropTypes.

Reply:
Though note that `theme` isn't actually required for consumers per se as the component is publicly exported `withTheme`.